tests/integrations/tests/test_openai.py

```
@@ -1579,16 +1603,936 @@ def test_40_text_completion_streaming(self, openai_client, test_config):
         assert chunk_count > 1, f"Streaming should have multiple chunks, got {chunk_count}"
 
         print(f"Success: Streamed haiku ({chunk_count} chunks): {content}")
+
+    # =========================================================================
+    # FILES API TEST CASES
+    # =========================================================================
+
+    @pytest.mark.parametrize(
+        "provider,model", get_cross_provider_params_for_scenario("batch_file_upload")
+    )
+    def test_41_file_upload(self, openai_client, test_config, provider, model):
```

Ruff (0.14.8)

Unused method argument: `openai_client`

(ARG002)